src/applications/personalization/profile/components/personal-health-care-contacts/Contact.jsx

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { capitalize } from 'lodash';
 
 // modeled after VA Profile's Health Benefit AssociatedPersonBio
 
+const CONTACT_TYPES = [
+  'Emergency Contact',
+  'Other emergency contact',
+  'Primary Next of Kin',
+  'Other Next of Kin',
+];
+
+const DESCRIPTIONS = {
+  [CONTACT_TYPES[0]]: 'The person we’ll contact first in an emergency.',
+  [CONTACT_TYPES[1]]:
+    'The person we’ll if your primary contact isn’t available.',
```

Comment:
TODO: Fix typo. Should read "The person we'll **contact** if your primary contact isn't available."